web/src/main/java/com/fitgoal/web/resources/NotificatorResource.java

```diff
@@ -0,0 +1,61 @@
+package com.fitgoal.web.resources;
+
+import com.codahale.metrics.annotation.Timed;
+import com.fitgoal.api.domain.Recipient;
+import com.fitgoal.api.NotificationService;
+import com.fitgoal.api.domain.UserVerification;
+
+import javax.inject.Inject;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@Path("/notifications")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+public class NotificatorResource {
+
+    private final NotificationService notificationService;
+
+    @Inject
+    public NotificatorResource(NotificationService notificationService) {
+        this.notificationService = notificationService;
+    }
+
+    @POST
+    @Path("/register")
+    @Timed
+    public Response register(@NotNull @Valid UserVerification userVerification) {
+        notificationService.register(userVerification);
+        return Response.ok().build();
```

Comment:
Do you want to return `200` intentionally? I'd rather make all the methods `void` and let Jersey return `204 (No Content)`.

Reply:
Refactored